### ISSUE TITLE
Missing Arduino.h (and other?) references

### ISSUE THREAD

**Issue by lonerzzz** [kuba2k2/libretuya#72]
When attempting to use libretuya in Platform.io with the latest release, I see that Arduino.h is unresolved. Please document/outline what is required in the platformio.ini file or elsewhere make it visible. I see that it exists in this project.

**Comment by lonerzzz** [comment]
Answering myself in case someone else see this. Just change the framework in the platformio.ini from beken-72xx-sdk to arduino

Feel free to add more detail.

**Comment by kuba2k2** [comment]
Yeah, the -sdk framework doesn't contain Arduino.h or any arduino related code.

**Comment by kuba2k2** [comment]
I'll keep it open to describe the framework differences in the docs.
